packages/library/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -23,9 +23,17 @@ import kotlinx.coroutines.flow.Flow
 import kotlin.reflect.KClass
 
 /**
- * This class represents the writeable state of a Realm file. The only way to modify data in a Realm is through
- * instances of this class. These are provided and managed automatically through either [Realm.write] or
+ * Represents the writeable state of a Realm file.
+ *
+ * The only way to modify data in a Realm is through instances of this class.
+ * These are provided and managed automatically through either [Realm.write] or
  * [Realm.writeBlocking].
+ *
+ * All objects created and/or obtained from the _mutable realm_ in a write-transaction are bound to
+ * the thread executing the transaction and all operations on the _mutable realm_ or on any of the
+ * objects must be done on the thread executing the transaction. Only exception are objects returned
```

Comment:
```suggestion
 * objects contained in that realm must execute on the thread executing the transaction. The only exception is objects returned
```

Clarification about objects in a particular realm and subject-verb agreement fix.